tensorflow/buildspec-graviton.yml

```diff
@@ -33,6 +37,20 @@
       tag_python_version: &TAG_PYTHON_VERSION py38
       os_version: &OS_VERSION ubuntu20.04
       tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
-      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton.ec2., *DEVICE_TYPE ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton., *DEVICE_TYPE ]
```

Comment:
Any reason for leaving out ec2 here (i.e. /Dockerfile.graviton.) but having ec2 in buildspec-graviton-2.7.yml definition file?

Reply:
because that does not exist in that folder. Dockerfile.graviton.yaml has both EC2 and SM images in it.